app/src/main/java/de/danoeh/antennapod/ui/screen/queue/QueueFragment.java

```diff
@@ -372,16 +379,21 @@ public boolean onContextItemSelected(MenuItem item) {
         }
 
         final int itemId = item.getItemId();
-        if (itemId == R.id.move_to_top_item) {
-            queue.add(0, queue.remove(position));
-            recyclerAdapter.notifyItemMoved(position, 0);
-            DBWriter.moveQueueItemToTop(selectedItem.getId(), true);
-            return true;
-        } else if (itemId == R.id.move_to_bottom_item) {
-            queue.add(queue.size() - 1, queue.remove(position));
-            recyclerAdapter.notifyItemMoved(position, queue.size() - 1);
-            DBWriter.moveQueueItemToBottom(selectedItem.getId(), true);
-            return true;
+        if (recyclerAdapter.inActionMode()) {
+            new EpisodeMultiSelectActionHandler(getActivity(), item.getItemId())
+                    .handleAction(recyclerAdapter.getSelectedItems());
```

Comment:
Is this actually called from here? I think it is only called in `floatingSelectMenu.setOnMenuItemClickListener`. This might be a leftover from the long-press menu you had before.